dependencydiff/mapping.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencydiff
+
+import (
+	"fmt"
+)
+
+// EcosystemOSV is a package ecosystem supported by OSV.
+type ecosystemOSV string
+
+// Data source: https://ossf.github.io/osv-schema/#affectedpackage-field
+const (
+	// The Go ecosystem.
+	goOSV ecosystemOSV = "Go"
```

Comment:
let's just call it `ecosystem`. The comment is enough without the need to bloat the variables names.
You can name each variable `ecosystemGo`, `ecosystemNpm`, etc

Reply:
done & resolved, thanks!